homeassistant/components/freebox/__init__.py

```diff
@@ -42,8 +48,18 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Freebox entry."""
-    router = FreeboxRouter(hass, entry)
-    await router.setup()
+    api = await get_api(hass, entry.data[CONF_HOST])
+    try:
+        await api.open(entry.data[CONF_HOST], entry.data[CONF_PORT])
+    except HttpRequestError as err:
+        _LOGGER.exception("Failed to connect to Freebox")
```

Comment:
We try to avoid logging before raising `ConfigEntryNotReady` as that log will be seen on every retry instead of just the first. Maybe it's enough to include the exception representation in the argument to `ConfigEntryNotReady`?

Reply:
Previously this bit of code was wrong I think (it would return instead of raising)
I think we can just drop the logging.